public/app/features/runtime/init.ts

```diff
@@ -0,0 +1,57 @@
+import { UrlQueryMap, PanelData } from '@grafana/data';
+import { getLocationSrv } from '@grafana/runtime';
+import { getDashboardSrv } from '../dashboard/services/DashboardSrv';
+import { getTimeSrv } from '../dashboard/services/TimeSrv';
+
+/**
+ * This will setup features that are accessible through the root window location
+ *
+ * This is useful for manipulating the application from external drivers like puppetter/cypress
+ *
+ * @internal and subject to change
+ */
+export function initWindowRuntime() {
+  (window as any).grafanaRuntime = {
+    /** Navigate the page within the currently loaded application */
+    load: (path: string, query?: UrlQueryMap) => {
```

Comment:
```suggestion
    updateLocation: (path: string, query?: UrlQueryMap) => {
```

Just to reduce the ambiguity of load